### ISSUE TITLE
Unable to use custom dns hook

### ISSUE THREAD

**Issue by foosinn** [acmesh-official/acme.sh#1571]
Steps to reproduce
------------------

Request a certificate with a custom dns hook like this:

```
/home/uwsgi/.acme.sh/acme.sh --issue -d example.com --dns ./test.sh
```

A DNS root is only applied if the script starts with `dns`

So here we are already failing if we are supplying just `/path/to/api/hook/file.sh`

Also `_findHook()` does not allow for a absolue path.

Debug log
-----------------

```
https://p.unigs.de/7JWvgtMD/+inline?
```

**Comment by Neilpang** [comment]
Yes, "absolue path" was first designed, but never got implemented yet.

you can rename you hook file and move it to `~/.acme.sh/` folder.

The `_findHook` function can find it.

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.

**Comment by neilpang** [comment]
Not a bug: hooks are looked up by name in the hook folders by design; place the custom script in ~/.acme.sh or dnsapi/ with a dns_ prefix.
